Nightmare/Ilgynoth.lua

```diff
@@ -105,6 +109,8 @@ if L then
 	L.deathglare_tentacle = -13190 -- Deathglare Tentacle
 	L.deathglare_tentacle_icon = 208697 -- Mind Flay icon
 
+	L.nightmare_ichor = -13186 -- Nightmare Inchor

```

Comment:
This can also be removed, there isn't a need for it, use the id directly in the option.